ariadne/playground.py

```diff
@@ -0,0 +1,60 @@
+PLAYGROUND_HTML = """
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset=utf-8/>
+  <meta name="viewport" content="user-scalable=no, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, minimal-ui">
+  <title>GraphQL Playground</title>
+  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/graphql-playground-react@1.7.0/build/static/css/index.css" />
+  <link rel="shortcut icon" href="//cdn.jsdelivr.net/npm/graphql-playground-react@1.7.0/build/favicon.png" />
+  <script src="//cdn.jsdelivr.net/npm/graphql-playground-react@1.7.0/build/static/js/middleware.js"></script>
+</head>
+
+<body>
+  <div id="root">
+    <style>
+      body {
+        background-color: rgb(23, 42, 58);
+        font-family: Open Sans, sans-serif;
+        height: 90vh;
+      }
+
+      #root {
+        height: 100%;
+        width: 100%;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+
+      .loading {
+        font-size: 32px;
+        font-weight: 200;
+        color: rgba(255, 255, 255, .6);
+        margin-left: 20px;
+      }
+
+      img {
+        width: 78px;
+        height: 78px;
+      }
+
+      .title {
+        font-weight: 400;
+      }
+    </style>
+    <img src='//cdn.jsdelivr.net/npm/graphql-playground-react/build/logo.png' alt=''>
+    <div class="loading"> Loading
+      <span class="title">GraphQL Playground</span>
+    </div>
+  </div>
+  <script>window.addEventListener('load', function (event) {
+      GraphQLPlayground.init(document.getElementById('root'), {
+        // options as 'endpoint' belong here
+      })
+    })</script>
+</body>
+
+</html>
+""".strip()
```

Comment:
Why not just putting a backslash like this:
```python
"""\
Hello
world\
"""
```

Instead of striping just for two `\n`?

Reply:
Personal preference - I don't like the `\` and startup performance hit from `.strip()` is minimal. ;)

Reply:
@rafalp do you have the button to mark the conversation as resolved? I don't why but I can no longer mark review as resolved, not sure if github changed the mechanism or if it's a bug or something.

Reply:
AFAIK you can't resolve comments, only changes requested, because repo can be setup to block unapproved merges.